tests/cross/src/test/scala/tests/highlight/DocumentHighlightSuite.scala

```diff
@@ -655,4 +655,18 @@ class DocumentHighlightSuite extends BaseDocumentHighlightSuite {
        |}""".stripMargin,
   )
 
+  check(
+    // Scala 2.12.x has a bug where the namePos points at `object`
```

Comment:
Tried working around it but it seems to be a lot of additional work, so I think I would leave it for now